test/e2e_node/util.go

```diff
@@ -231,8 +226,8 @@ func updateKubeletConfig(ctx context.Context, f *framework.Framework, kubeletCon
 
 	framework.ExpectNoError(e2enodekubelet.WriteKubeletConfigFile(kubeletConfig))
 
-	ginkgo.By("Starting the kubelet")
-	startKubelet()
+	ginkgo.By("Restarting the kubelet")
+	restartKubelet()
 	waitForKubeletToStart(ctx, f)
```

Comment:
do you need this `waitForKubeletToStart`?

Reply:
Thanks for pointing out! Removed.